Taskfile.yml

```
@@ -0,0 +1,145 @@
+version: '3'
+env:
+  vars:
+    - PYTHONPATH: {{ .ROOT_DIR }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)